CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ option(PYBINDINGS "Create makefiles for building python bindings" OFF)
 option(SKIP_DESKTOP "Skip building of desktop application" OFF)
 option(BUILD_MAPSHOT "Build mapshot tool" OFF)
 option(USE_PCH "Use precompiled headers" OFF)
+option(USE_CCACHE "Cache object files with ccache" OFF)
```

Comment:
one second thought, maybe add USE_GOLD too (also off by default) in case it doesn't work for somebody?

Reply:
ld.gold is part of binutils since 2008, and, to my knowledge, the only known unsolvable problem it has is linking kernel modules.

ld.gold might reveal some other problems, like forgetting to explicitly link to pthread or rt, according to https://stackoverflow.com/questions/3476093/replacing-ld-with-gold-any-experience but then those problems should be fixed instead, I believe.

So unless you insist on this option, I would prefer not to add it